tests/cxx/tests/DREAM/RunawayFluid.cpp

```diff
@@ -197,8 +199,8 @@ bool RunawayFluid::CompareEceffWithTabulated(){
     REFluid = GetRunawayFluid(cq,N_IONS2, Z_IONS2, ION_DENSITY_REF, T_cold,B0,nr);
     Eceff3 = REFluid->GetEffectiveCriticalField(0);
 
-    real_t TabulatedEceff1 = 8.88124;
-    real_t TabulatedEceff2 = 8.00712;
+    real_t TabulatedEceff1 = 8.86778;
+    real_t TabulatedEceff2 = 7.99316;
```

Comment:
it says that the test fails in the latest commit. Update these values again?

Reply:
Eeeeeh yes, forgot after the last bug fix...